tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": ["esnext"],
+    "lib": ["esnext", "dom"],
```

Comment:
why do we need to add `dom` here? i remember running into similar issue and fixed it before with a dependency changed, but forgot what caused it

Reply:
Without it tsc complains everytime we use fetch. This was more a hack because of course zkitter doesn't run in the browser so we shouldn't need the DOM lib.
Otherwise I think we need to polyfill fetch

Reply:
If we upgrade to Node 18 we won't need any polyfill anymore as Node > 18 has the fetch API. But it would probably break a couple of dependencies.